lms/tasks/roster.py

```diff
@@ -21,7 +21,7 @@
 LAUNCHED_WINDOW = timedelta(hours=24)
 """How recent we need to have seen a launch from a course/assignment before we stop fetching rosters for it."""
 
-ROSTER_REFRESH_WINDOW = timedelta(hours=24 * 7)
+ROSTER_REFRESH_WINDOW = timedelta(hours=24 * 5)
```

Comment:
We'll see how this change goes and likely go lower.

We'll have to tweak this dashboard to consider the new parameter.
https://report.hypothes.is/dashboard/37-rostering-dashboard


After changing the query there the column with "backlog" values will increase. Once it's back to zero consistenly we can decrease the value again.